.pipelines/cni/cilium/cilium-scale-test.yaml

```diff
@@ -0,0 +1,228 @@
+                cd test/scale/connectivity
+                chmod +x test-connectivity.sh
+                ./test-connectivity.sh --num-scale-pods-to-verify=${NUM_SCALE_PODS_TO_VERIFY} --max-wait-for-initial-connectivity=600 --max-wait-after-adding-netpol=120
+            name: "TestNetworkPolicies"
+            displayName: "Network Policies Connectivity Test"
+  - stage: scale_for_load_tests
+    displayName: "Scale for load tests"
+    jobs:
+      - job: deploy_service
+        pool:
+          name: "$(BUILD_POOL_NAME_DEFAULT)"
+        steps:
+          - task: AzureCLI@1
+            inputs:
+              azureSubscription: $(TEST_SUB_SERVICE_CONNECTION)
+              scriptLocation: "inlineScript"
+              scriptType: "bash"
+              addSpnToEnvironment: true
+              inlineScript: |
+                az aks get-credentials --resource-group ${CLUSTER} --name ${CLUSTER}
+                cd test/scale
+                chmod +x test-scale.sh
+                ./test-scale.sh --max-kwok-pods-per-node=0 --num-kwok-deployments=0 --num-kwok-replicas=0 --max-real-pods-per-node=${REAL_PODS_PER_NODE_LB} --num-real-deployments=${NUM_REAL_DEPLOYMENTS_LB} --num-real-replicas=${NUM_REAL_REPLICAS_LB} --num-network-policies=0 --num-unapplied-network-policies=0 --num-unique-labels-per-pod=0 --num-unique-labels-per-deployment=5 --num-shared-labels-per-pod=3 --num-real-services=${NUM_REAL_SVC_LB} --delete-labels --real-pod-type=nginx
+            name: "TestLBServices"
+            displayName: "Scale for load tests"
+  - stage: benchmark_testing
+    displayName: "Run apachebench test"
+    jobs:
+      - job: apachebench_test
+        pool:
+          name: "$(BUILD_POOL_NAME_DEFAULT)"
+        steps:
+          - task: AzureCLI@1
+            inputs:
+              azureSubscription: $(TEST_SUB_SERVICE_CONNECTION)
+              scriptLocation: "inlineScript"
+              scriptType: "bash"
+              addSpnToEnvironment: true
+              inlineScript: |
+                az aks get-credentials --resource-group ${CLUSTER} --name ${CLUSTER}
+                echo "Deploy apachebench pod and run test"
+                cd hack/manifests
+                kubectl apply -f apache.yaml
+                echo "wait for pod to become ready"
+                kubectl rollout status deployment apachebench --timeout=30s
+                kubectl get pod -owide
+                mkdir results
+                cd results
+                AB_POD=$(kubectl get pod -l app=apachebench | grep apachebench | awk '{print $1}')
+                kubectl exec -it $AB_POD -- ab -n ${AB_REQUESTS} -c ${AB_CONCURRENCY} -r http://real-svc-00001.scale-test/ >> "ab_${AB_REQUESTS}requests_${NUM_REAL_REPLICAS_LB}kpods.log"
+                echo "running k top node" 
+                kubectl top node >> "node_cpu_and_mem.log"
+                echo "running k top pod"
+                kubectl top pod -A | grep cilium >> "pod_cpu_and_mem.log"
```

Comment:
lets get pod cpu and mem at each step before and after network policy scale and after lb scale